plugins/connection/aws_ssm.py

```diff
@@ -540,35 +541,34 @@ def _initialize_ssm_client(self, region_name: Optional[str], profile_name: str)
             None
         """
 
-        self._vvvv("SETUP BOTO3 CLIENTS: SSM")
+        self.verbosity_display(4, "SETUP BOTO3 CLIENTS: SSM")
         self._client = self._get_boto_client(
             "ssm",
             region_name=region_name,
             profile_name=profile_name,
         )
 
-    def _display(self, f: Any, message: str) -> None:
+    def verbosity_display(self, level: int, message: str) -> None:
```

Comment:
The function docs say there's a default value of 1, but that's not represented in the function signature.

Reply:
Good catch! I meant to remove that default value from the docstring.